### PR TITLE
Add sentry

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

File: airflow/bin/cli.py [hunk 1]
```diff
@@ -58,13 +58,39 @@
...
+    except ImportError:
+        pass
+    else:
+        register_logger_signal(client)
+        register_signal(client, ignore_expected=True)
+
+    return client
+
+
+sentry = setup_sentry()
```
Comment (mattrobenolt) [inline airflow/bin/cli.py:91]: Why even assign this to a variable?

**Review by mattrobenolt — APPROVED** [review]

File: airflow/bin/cli.py (hunk 1 above)
Comment (dcramer) [reply airflow/bin/cli.py:91]: figured in case we need to reference it (not that we do)

also it prevents gc

**Comment by s7anley** [comment]
Any chance this will get into upstream?

**Comment by dcramer** [comment]
It's an Apache project, so the pessimist in me says not likely. We at least have no intention to go through the process but you're welcome to try getting it in. 

It's also possible it's not needed and there's a better system that could be built for error handling.

**Comment by s7anley** [comment]
Ok, thanks.

**Pull request merged**